src/core/providers/canvas-schema/canvas.const.ts

```diff
@@ -11,6 +11,8 @@ const ROW_HEIGHT = FONT_SIZE + ROW_PADDING; // Height of the table line
 const HORIZONTAL_LEFT_EXTENSION = 20; // Horizontal extension for the relation line 1:1
 const HEADER_TITLE_GAP = 15;
 const CANVAS_PADDING = 100; // Represents the additional space around the canvas, we use it to add a space for the tables when we export the schema
+const MAX_PLACEMENT_ATTEMPTS = 20; // Maximum number of attempts to place a table without overlapping
+const TABLE_SHIFT_DISTANCE = 30; // Distance to shift the table when it overlaps with another table
```

Comment:
There's something strange with this updates, they don't belong to the case, maybe you need to update from latest vnext cut?